pandas/tests/frame/test_reductions.py

```diff
@@ -1686,6 +1686,48 @@ def test_minmax_extensionarray(method, numeric_only):
     tm.assert_series_equal(result, expected)
 
 
+def test_mad_nullable_integer():
+    # GH#33036
+    df = DataFrame(np.random.randn(100, 4).astype(np.int64))
+    df2 = df.astype("Int64")
```

Comment:
Should this work with the `any_signed_int_ea_dtype` fixture?

Reply:
good idea, updated